lotr-app/pages/volumes/[slug].js

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from "next/router";
+import { volumes } from "@/public/lib/data";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function VolumeDetail() {
+  const router = useRouter();
+  const { slug } = router.query;
+
+  const volume = volumes.find(() => slug === volume.slug);
+  if (!volume) return;
+  const { books } = volume;
```

Comment:
Here instead of destructuring volume to get the books, you could just use volume.books on the line 26. Both works, you just would avoid this extra line.